subjects/findprevprime/README.md

```diff
@@ -21,15 +21,11 @@ Here is a possible program to test your function :
 ```go
 package main
 
-import (
-	"fmt"
-
-	"piscine"
-)
+import "fmt"
 
 func main() {
-	fmt.Println(piscine.FindPrevPrime(5))
```

Comment:
May I ask why this one was modified instead of creating a new one with a new ref like the rest?

If it doesn't require any action, feel free to resolve the conversation :slightly_smiling_face:

Reply:
This one is only used in the exams, so it can be changed in the current readme